stock_package_info/models/stock_quant_package.py

```diff
@@ -19,10 +19,6 @@ class StockQuantPackage(models.Model):
         string='Pickings',
         compute='_compute_picking_ids',
     )
-    product_packaging_id = fields.Many2one(
```

Comment:
This is worth noting. I honestly don't know what this column was for, but there's another `product.packaging` relation that already exists for the package that is being referenced. Maybe someone else knows what this is for, but I'm axing it otherwise.